yt_dlp/extractor/rai.py

```diff
@@ -339,6 +342,7 @@ def _real_extract(self, url):
             'episode': media.get('episode_title'),
             'episode_number': int_or_none(media.get('episode')),
             'subtitles': subtitles,
+            'edit_year': original_year,
```

Comment:
this is not a valid field name 
see common.py for valid fields

Reply:
modified_date may be suitable here